week_3/15_apr/dom.js

```diff
@@ -0,0 +1,24 @@
+const para = document.querySelector('p')
+console.log(para);
+
+console.log(para.innerText);
+
+para.innerText = "Hi"
+
+const paras = document.querySelectorAll('p')
```

Comment:
@codal-kcolaco what is diffrent btw querySelectorAll and querySelector?

Reply:
@abhi-codal querySelector selects a single element, while querySelectorAll is used to select multiple elements together. The above code returns a list of all elements with p tags.